language-examples/python/vectara-rest/rest_api_key_queries.py

```diff
@@ -25,7 +25,8 @@ def _get_query_json(customer_id: int, corpus_id: int, query_value: str):
 
```

Comment:
I like how you condensed constructing the query in `language-examples/python/vectara-rest/rest_query.py`. Can you please do that here too? Thanks.

Reply:
Done.